idp-core/src/main/java/io/mosip/idp/core/dto/OAuthDetailResponse.java

```diff
@@ -20,6 +20,6 @@ public class OAuthDetailResponse {
     private List<String> authorizeScopes;
     private List<String> essentialClaims;
     private List<String> voluntaryClaims;
-    private Map<String, String> configs;
+    private Map<String, Object> configs;
```

Comment:
why is this object ?, How would UI know which type to cast that ?

Reply:
UI will be fetching only the required config key, so assuming that they will know the value type.